src/docker/ros2_object_detection/Dockerfile

```diff
@@ -9,8 +9,6 @@ RUN apt-get update && apt-get install -y curl gnupg2 lsb-release software-proper
 RUN curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key -o /usr/share/keyrings/ros-archive-keyring.gpg
 RUN echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(. /etc/os-release && echo $UBUNTU_CODENAME) main" | tee /etc/apt/sources.list.d/ros2.list > /dev/null
 
-RUN wget -c https://download.openmmlab.com/mmdetection/v2.0/yolo/yolov3_mobilenetv2_320_300e_coco/yolov3_mobilenetv2_320_300e_coco_20210719_215349-d18dff72.pth -O "/root/mmdetection/configs/yolo/yolov3_mobilenetv2_320_300e_coco_20210719_215349-d18dff72.pth"
```

Comment:
How is the pth file downloaded?

Reply:
Through the mim as in the README. 
E.g.: 
mim download mmdet --config yolov3_mobilenetv2_8xb24-320-300e_coco --dest .
The mim tool will download both files, *.py and *.pth

Reply:
Ok, but it is not in the Dockerfile...

Reply:
It is:
[src/docker/era_5g_netapp_base_mmcv/gpu.Dockerfile](https://github.com/5G-ERA/Reference-NetApp/pull/59/files#diff-9d32eda40b113e2aab6620a43c37678cd29949540d4ffe5a33a61573c1e1f0c8)
and:
https://github.com/5G-ERA/Reference-NetApp/pull/59/files#diff-70ec977244601c98db30beeac823949927f99cdcd40a297aa341c102443a4f1b